src/PrestaShopBundle/Resources/views/Admin/Layout/default_layout.html.twig

```diff
@@ -38,7 +38,16 @@
       {% block extra_stylesheets %}{% endblock %}
     {% endblock %}
   </head>
-  <body class="lang-{{ ps.isoUser }}{% if ps.isRtlLanguage %} lang-rtl{% endif %} {{ ps.controllerName|escape|lower }}{% if ps.menuCollapsed %} page-sidebar-closed{% endif %}{% if ps.multiShop|default(false) %} multishop-enabled{% endif %}{% if ps.debugMode|default(false) %} developer-mode{% endif %}"
+
+  {# Construct CSS classes based on version numbers #}
+  {% set major_version = constant('PrestaShop\\PrestaShop\\Core\\Version::MAJOR_VERSION') %}
```

Comment:
I think you could ask core developer to do it cleanly on the back end side of things.

Reply:
Ok, I have asked @jolelievre what he thinks about that.

Reply:
We indeed have something similar already available `{{ ps.version }}` But it contains the whole version, however careful as @kpodemski said this one shouldn't be used in the login page as we try to limit the number of elements that help defining the PrestaShop version on public pages

Inside the back office it should be fine we could add something like `{{ ps.majorVersion }}` and so on for minor and patch versions. But I would need to have more details about your need Why do you need a body class that depends on the version I didn't see any reference in the PR (I may have missed them), is it required in the UI kit?